README.md

```diff
@@ -3,10 +3,9 @@
 API driven OpenShift 4 cluster provisioning and management.
 
 Hive is an operator which runs as a service on top of Kubernetes/OpenShift.
-The Hive service can be used to provision and manage OpenShift clusters.
+The Hive service can be used to provision and perform initial configuration required for fresh installed OpenShift clusters.
```

Comment:
I think "initial configuration" implies "fresh installed", I would just go with "and perform initial configuration of OpenShift clusters". That or s/fresh/freshly.